Speekins/whats-cookin
# Add/new tests

#### What's this PR do?
This merge will adds to the test suite in order to test new functionality added to the classes throughout part II of this project
#### Where should the reviewer start?
- Pull down branch
- See test for removing recipes for `User` class in `User-test.js`
- See tests for no repeat ingredients and properly compiled amounts (from repeat ingredients) for `Recipe` class in `Recipe-test.js`
- See test for properly formatted unit types for `Ingredient` class in `Ingredient-class.js`
#### How should this be manually tested?
- Run `npm test` from the `whats-cookin` repository root to see that all tests are unskipped and passing
#### Any background context you want to provide?
Part II of this project added a lot of functionality around access and formatting data from the server. This functionality was added throughout the project, but tests weren't properly in place before this PR.
#### Screenshots
![image](https://user-images.githubusercontent.com/74210902/200443206-7fae5039-4fd7-4f8f-81c5-fc8eaa64433b.png)
![image](https://user-images.githubusercontent.com/74210902/200443256-a6c00274-91b8-4378-a7e3-a93f62a778ec.png)
![image](https://user-images.githubusercontent.com/74210902/200443390-28f27836-83e9-4519-9e42-8bd494f7e68b.png)
